### PR TITLE
Add ask, aggregate and compute query scenarios

### PR DESCRIPTION
**Pull request by aelred**
@mikonapoli I wrote these in the more explicit style you recommended.

The graph itself is still just the Pokemon graph, but we can change that later.

### CODE REVIEW

File: features/graql-queries.feature [hunk 1]
```diff
@@ -28,13 +28,23 @@ Feature: Graql Queries
...
     Scenario: Match Query With Non-Empty Response
-		When The user issues a match query which should have results
-    	Then Return a response with matching concepts
+        When The user issues a match query which should have results
+        Then Return a response with matching concepts
+
+
+    Scenario: Ask Query With False Response
+        When The user executes `match $x has name "Chicken"; ask;`
+        Then Return `False`
```
Comment (mikonapoli) [inline features/graql-queries.feature]: I do not think that "Return false" is a good idea.

Slightly better:
Then return a non-empty response
And the response should contain "False"

Slightly betterer:
Then a "False" response is returned

My issue is that the protagonist of the scenario is the user, and it's not the user returning the response. Also, we want to make sure that the response is returned, but we do not care _who_ returns it (and we would according to the scenario as written).

File: features/graql-queries.feature [hunk 2]
```diff
@@ -28,13 +28,23 @@ Feature: Graql Queries
...
 
     Scenario: Match Query With Non-Empty Response
-		When The user issues a match query which should have results
-    	Then Return a response with matching concepts
+        When The user issues a match query which should have results
+        Then Return a response with matching concepts
+
+
+    Scenario: Ask Query With False Response
+        When The user executes `match $x has name "Chicken"; ask;`
```
Comment (mikonapoli) [inline features/graql-queries.feature]: Let us be consistent: issues or executes? One of the objectives of cucumber tests is to develop a common language.
Also: how do we no there is no chicken in the graph? Either we make the background more specific (preferred by me) or we make this more generic

Comment (aelred) [inline features/graql-queries.feature]: I agree on the chicken thing: I'll try specifying the ontology + data in the scenario. I prefer "execute", but for now I'll keep consistent with the old terminology (we can change the whole lot later).

File: features/graql-queries.feature (hunk 1 above)
Comment (aelred) [inline features/graql-queries.feature]: Sounds good!

Comment (aelred) [inline features/graql-queries.feature]: Actually maybe not quite perfect. It doesn't read too clearly with more complex responses (such as `match` responses):
> Then a "{'x': {'id': '4567', 'value': 'Chicken'}}" response is returned
Maybe we could do:
> Then the response is "{'x': {'id': '4567', 'value': 'Chicken'}}"
or something

**Comment by aelred** [comment]
@mikonapoli please re-review!

**Pull request merged**